.github/workflows/x-create-tags.yml

```diff
@@ -0,0 +1,69 @@
+
+name: X Create tags
+
+on:
+  workflow_call:
+    inputs:
+      gh-org:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      branch:
+        required: false
+        type: string
+        default: main
+      set-versions:
+        required: false
+        type: boolean
+        default: false
+    secrets:
+      GH_TOKEN:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-tags:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repository:
+          - keycloak-client
+
+    steps:
+      - name: Checkout repository
```

Comment:
If this is supposed to be a re-usable action, then the workflow that uses it should be responsible for checking out the source code. It should not be repeated here.